packages/elements/src/components/layout/layout.stories.mdx

```diff
@@ -97,3 +71,121 @@ For a complete example of how to lay out a page, see below where we have combine
   story={['Complete Combined Example']}
   label="Complete Combined Example Rendered HTML markup"
 />
+
+## Atomic design
+
+Atomic design is a design system methodology for defining the structure or scale of a set of components. A simple explaination is as follows: the smallest component is defined as an `Atom`. A collection of Atoms is defined as a `Molecule`. A collection of Molecules is defined as an `Organisum`.
```

Comment:
So I was thinking you would add an Atom and an Organism as components - not sure if it makes sense but just a suggestion